### PR TITLE
Update 1.25 release schedule

### PR DESCRIPTION
**Pull request by Verolop**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
Update schedule for 1.25.0-alpha.1, 1.25.0-alpha.2, 1.25.0-alpha.3, 1.25.0-beta.0, 1.25.0-rc.0, 1.25.0-rc.1

#### Special notes for your reviewer:
open to proposals and changes

### CODE REVIEW

File: releases/release-1.25/README.md [hunk 1]
```diff
@@ -57,34 +57,36 @@ The 1.25 release cycle is as follows:
...
 | Release blog ready to review                                  | Comms / Docs | 01:00 UTC Wednesday 10th August 2022 / 02:00 BST Wednesday 10th August 2022 / 18:00 PDT Tuesday 9th August 2022 | week 12  | |
 | Major Themes complete                                         | Release Notes Lead | Tuesday 16th August 2022                                                                                        | week 13  | |
 | Docs complete — All PRs reviewed and ready to merge           | Docs Lead | Tuesday 16th August 2022                                                                                        | week 13  | |
 | Feature blogs ready to review                                 | Enhancement Owner / SIG Leads | Tuesday 16th August 2022                                                                                        | week 13  | |
+| 1.25.0-rc.1 released                                          | Branch Manager | Tuesday 16th August 2022                                                                                                      | week 13  | |
+
```
Comment (cpanato) [inline releases/release-1.25/README.md:88]: nit: extra line can be removed

File: releases/release-1.25/README.md [hunk 2]
```diff
@@ -57,34 +57,35 @@ The 1.25 release cycle is as follows:
 | Schedule finalized                                            | Lead | Friday 27th May 2022                                                                                            | week 1   | |
 | Team finalized                                                | Lead | Friday 27th May 2022                                                                                            | week 1   | |
 | Start Release Notes Draft                                     | Release Notes Lead | Tuesday 7th June 2022                                                                                           | week 3   | |
+| 1.25.0-alpha.1 released                                       | Branch Manager | Wednesday 15th June 2022                                                                                             | Week 4   | |
+
```
Comment (saschagrunert) [inline releases/release-1.25/README.md:61]:
```suggestion
```

This breaks the table, so we have to remove this line :blush:

**Review by saschagrunert — COMMENTED** [review]
Just a nit, otherwise LGTM

File: releases/release-1.25/README.md (hunk 2 above)
Comment (Verolop) [reply releases/release-1.25/README.md:61]: thank you! I obviously had a very messy markdown 😅

**Review by saschagrunert — APPROVED** [review]
Thanks!

**Pull request merged**